website/docs/reference/artifacts/sl-manifest.md

```diff
@@ -7,26 +7,24 @@ sidebar_label: "Semantic manifest"
 pagination_next: null
 ---
 
+**Produced by:**  Any command that parses your project. This includes all commands _except_ [`deps`](/reference/commands/deps), [`clean`](/reference/commands/clean), [`debug`](/reference/commands/debug), and [`init`](/reference/commands/init).
```

Comment:
Does this require a [manifest version](https://schemas.getdbt.com/dbt/manifest/v12/index.html) similar to the manifest.json?

Reply:
it does! however i've asked about that [here](https://github.com/dbt-labs/docs.getdbt.com/issues/4369#issuecomment-2402719709) and [here](https://dbt-labs.slack.com/archives/C050Z6UP4RK/p1728488691096269) bc semantic manifest is missing from the schema.getdbt.com site as well. once i get guidance, I'll add it myself however i didn't want to block this pr for that.